src/app/common/services/validate-form.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { AbstractControl } from '@angular/forms';
+
+@Injectable({ providedIn: 'root' })
+export class ValidateService {
+
+  constructor() {}
+
+  public emailValidator(control: AbstractControl): { [key: string]: boolean } | null {
+    const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    if (!re.test(String(control.value).toLowerCase()) && control.value.trim() !== '' ) {
```

Comment:
why use `toLowerCase()` function ? we don't do anything to change input email, if do that, regex doesn't make sense anymore.